stravalib/unithelper.py

```diff
@@ -42,6 +42,10 @@ def unit(self):
         warn_units_deprecated()
         return str(self.q.units)
 
+    def __eq__(self, other):
```

Comment:
Added this to compare quantities in unit tests. Eventually this will be deprecated and we can use the rich comparison methods from Pint.